packages/core/src/StateMachine.ts

```diff
@@ -374,16 +379,15 @@ export class StateMachine<
 
     return macroState;
   }
+
   public start(
     state: State<TContext, TEvent, TResolvedTypesMeta>,
     actorCtx: ActorContext<TEvent, State<TContext, TEvent, TResolvedTypesMeta>>
-  ): State<TContext, TEvent, TResolvedTypesMeta> {
+  ): void {
```

Comment:
This change should be documented with a changeset

Reply:
Don't know if it makes much sense (this isn't a change from v4 since it's a new method) but I'll add one.